webui/test/e2e/common/viewParquetObject.spec.ts

```diff
@@ -39,6 +39,7 @@ test.describe("Object Viewer - Parquet File", () => {
         }
         await loginPage.doLogin(credentials.accessKeyId, credentials.secretAccessKey);
         await repositoriesPage.goto();
+        await page.waitForURL(/.*\/repositories/);
```

Comment:
I suggest adding this to the `doLogin()` function, since it represents the actual end of the `doLogin()` action.
It can then be removed from the other places using this `waitForURL()`.

And `await repositoriesPage.goto(); await repositoriesPage.goToRepository(TEST_REPO_NAME);` are not needed here - 
It can be replaced with `repositoryPage.goto(TEST_REPO_NAME);` after it's initialized.